src/components/BarcodeScanner/BarcodeScannerContainer.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Vibration,StyleSheet, Text, View } from 'react-native';
+import { BarCodeScanner, Permissions } from 'expo';
+
+export default class App extends React.Component {
```

Comment:
The name of this component is set to App but the suggestion below might be better:

```suggestion
export default class BarcodeScannerContainer extends React.Component {
```